DataFormats/Detectors/TRD/include/DataFormatsTRD/TrackTRD.h

```diff
@@ -0,0 +1,35 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TrackTRD.h
+/// \author David Rohr
+
+#ifndef O2_DATAFORMATS_TRACK_TRD_H
+#define O2_DATAFORMATS_TRACK_TRD_H
+
+#include "GPUTRDTrack.h"
+
+namespace o2
+{
+namespace trd
+{
+using TrackTRD = o2::gpu::GPUTRDTrack;
+} // namespace trd
```

Comment:
It is actually in the linkdef, but it is in the linkdef for the GPU, not for TRD, since the type is actually `o2::gpu::GPUTRDTrack_t < o2::gpu::trackInterface < o2::gpu::GPUTRDO2BaseTrack>>`, everything else are only using aliases.

I agree for the ClassDef. I am actually wondering why it is not there, didn't it work to store it in ROOT files in AliRoot? In any case, you can just add the `ClassDefNV` without protection, the inclusion of the `GPUCommon` headers will take care of that.

Reply:
Actually for storing the TRD tracks in AliRoot we converted them to `AliESDtrack` in the `AliHLTGlobalEsdConverterComponent.cxx:L1276`